app/trackedRoute.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
 import { Route } from 'react-router-dom'
 import ReactGA from 'react-ga'
+import config from 'config'
 
 export default class TrackedRoute extends React.Component {
   trackPage(page) {
+    document.title = config.titles[page.split('/')[1]]
```

Comment:
I don't feel this is the correct function to put this in, since this function is to do with page tracking. Instead, I think this is something that can happen on `render` or on `componentDidMount`.

Reply:
I agree on that, it shouldn't be on the trackPage function, render should be the place.